rsocket-core/src/main/java/io/rsocket/RSocketClient.java

```diff
@@ -490,8 +493,7 @@ private void handleFrame(int streamId, FrameType type, ByteBuf frame) {
             LimitableRequestPublisher sender = senders.get(streamId);
             if (sender != null) {
               int n = RequestNFrameFlyweight.requestN(frame);
-              sender.increaseRequestLimit(n);
-              sendProcessor.drain();
+              sender.request(n >= Integer.MAX_VALUE ? Long.MAX_VALUE : n);
```

Comment:
I'm trying to remember where we specify this rule?  Integer.MAX_VALUE == Inifinite?  Reactor? Reactive Streams? RSocket?

Reply:
https://github.com/rsocket/rsocket-java/blob/develop/rsocket-core/src/main/java/io/rsocket/RSocketServer.java#L469

Reply:
I guess what I'm asking is whether this rule needs to migrate into the spec

Reply:
If it is not clarify that Int. Max Value should be considered as unbounded stream, then we have to mention that